src/WebJobs.Extensions.DurableTask/Bindings/ObjectValueProvider.cs

```diff
@@ -38,7 +38,7 @@ public Task<object> GetValueAsync()
 
         public string ToInvokeString()
         {
-            return this.value.ToString();
+            return (this.value != null) ? this.value.ToString() : null;
```

Comment:
A more concise fix would be to do `return this.value?.ToString();`, but what you have here is fine too. :)